ci/python_build_sdist.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# build the sdist file, and try installing it
+
+set -euo pipefail
+
+BASE=$(pwd)/build/
+
+mkdir -p $BASE
```

Comment:
Why is it `"$VENV"`, but `$BASE`? `"$(pwd)"` could include spaces…

Reply:
100% right, thanks.